python/mxnet/contrib/onnx/onnx2mx/_op_translations.py

```diff
@@ -505,6 +529,30 @@ def exponent(attrs, inputs, proto_obj):
     """Elementwise exponent of input array."""
     return 'exp', attrs, inputs
 
+def _cos(attrs, inputs, proto_obj):
```

Comment:
why _cos, _sin for example need to start with leading underscore? in python they are under math namescope, so I guess there's no conflict

Reply:
there will be a namespace conflict if there is an import math statement. probably better practice to is the leading underscore for names that are very common and prevent namespace collision.